demos/browser/app/meeting/meeting.html

```diff
@@ -198,6 +198,7 @@ <h1 class="h3 mb-3 font-weight-normal text-center">Select devices</h1>
 
     <div id="flow-meeting" class="flow" style="position:absolute;left:15px;top:0;bottom:55px;right:15px">
       <div class="text-muted" style="position:fixed;left:3px;bottom:3px" id="video-uplink-bandwidth"></div>
+      <div class="text-muted" style="position:fixed;left:35%;bottom:3px" id="meeting-id-number"></div>
```

Comment:
Instead of left:35%, can we use margin:auto, width:40%, and text-align:center to center the div and the text?

Reply:
width:40% will let the meeting id to occupy only 40% of the width of the active browser, thus overlapping the uplink text on the left

Reply:
Alternative to left 35%, we can do width:100% and text-aligned to center